v1/message.go

```diff
@@ -37,6 +37,8 @@ type Content struct {
 	ChatID     string `json:"chat_id"`
 	Text       string `json:"text"`
 	ResponseTo string `json:"response-to"`
+	// The ENS name of the sender
+	Name string `json:"name"`
```

Comment:
If it's the ENS name, why not call the attribute `ENSName` to make it clear?
Also, keep formatting the same, add the comment at the end of line.

Reply:
While I agree with you, currently I'd like to keep the this inline with how it is in the protocol (it is `name` there), I would revisit once this part of the codebase settles, and we can go through all the fields (I don't think this whole field should be a nested map, it's a map only for historical reasons).

Reply:
I see, so you want to keep naming the same as in protocol spec, makes sense.

So yeah, just fix the formatting.